collect_app/src/main/java/org/odk/collect/android/widgets/RatingWidget.java

```diff
@@ -101,6 +101,7 @@ public IAnswerData getAnswer() {
     @Override
     public void clearAnswer() {
         binding.ratingBar1.setRating(0.0F);
+        widgetValueChanged();
```

Comment:
Also good to add tests for the behaviour changes in a610987094d967a1b7348eca7540253164783ae8

Reply:
Done.